src/main/java/org/apache/sling/servlets/get/impl/DefaultGetServlet.java

```diff
@@ -291,7 +279,7 @@ protected void doGet(SlingHttpServletRequest request,
                 request.getResource().getPath(), "No resource found");
         }
 
-        Servlet rendererServlet;
+        Renderer rendererServlet;
```

Comment:
Renaming the variable here would make sense as well.

Reply:
good point